doc/nrf/known_issues.rst

```diff
@@ -1661,7 +1671,7 @@ NCSDK-9992: Multiple extra CMake options applied as single option.
 Toolchain
 *********
 
-.. rst-class:: v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0
+.. rst-class:: v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-1 v1-5-0
```

Comment:
@joerchan  should this be removed?

Reply:
It's a problem that will never go away, the two versions mentions are broken. Is there a way to list it as a known_issue without listing affected versions.

The workaround would better if it said to use the version of the toolchain that we recommend for the specific version.

Reply:
Updated workaround.

There's no way to list a known issue without the affected versions. Besides, with no affected version, we're talking limitation, aren't we?

What we can do is to move this known issue as a note under this table: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_recommended_versions.html#required-tools and then propagate the note to other pages, for example: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/tfm/tfm_hello_world/README.html But this will give it less visibility than in the known issues, I guess.

Reply:
> There's no way to list a known issue without the affected versions. Besides, with no affected version, we're talking limitation, aren't we?

Not really, it is a known issue on versions that we don't provide. Moving it to limitations doesn't really make sense in my opinion, but if that fits better with our system it is a possebility.

I think it is best to move it as a note, we are documenting that we only recommend one specific version:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_installing.html#install-the-gnu-arm-embedded-toolchain

> To be able to cross-compile your applications for Arm targets, you must install version 9-2019-q4-major of the GNU Arm Embedded Toolchain.
>
> Important
>
>Make sure to install the toolchain version that is mentioned above. Other toolchain versions might not work with this version of the nRF Connect SDK. Similarly, other versions of the nRF Connect SDK might require a different toolchain version.

Reply:
The issue is now moved to a note. See the separate commit.